apps/explorer/lib/explorer/chain/address/token.ex

```diff
@@ -16,7 +16,7 @@ defmodule Explorer.Chain.Address.Token do
   import Ecto.Query
 
   alias Explorer.{Chain, PagingOptions}
-  alias Explorer.Chain.{Address, Address.TokenBalance, Hash}
+  alias Explorer.Chain.{Address, Address.CurrentTokenBalance, Hash}
```

Comment:
I know it was like that before, but WDYT about moving this `Address.CurrentTokenBalance ` alias that has two levels to another line? It looks a bit cluttered to me.

```suggestion
  alias Explorer.Chain.{Address, Hash}
  alias Explorer.Chain.Address.CurrentTokenBalance
```

Reply:
Done, thanks!